### DISCUSSION TITLE
大學了，大家還習慣嗎

### DISCUSSION THREAD

**Discussion by leotsai940914** [1101-GS4719-C/Python-Community-GS4719-C#2104] [Q&A]
終於來到了大學，對我而言，我很快就適應了。我覺得這大概得益於我的高中外宿生活。什麼都該自己來，順利跨過了成人的第一道坎，我們享有了更大的自我選擇權。

但與高中每週都能回家不同，這一次可能不再能以週為單位了，一個月，兩個月，三個月。說實在，當後車窗外熟悉的風景越來越遠，難免是感傷的，任誰其實都不免想多看個幾眼，可惜車就跟時間一樣只能前進不能後退了。

不知道各位最近如何，至少我目前生活的很滿意。

**Comment by MingWei174** [comment]
上了大學之後,不能每天和父母面對面聊天只能透過簡訊噓寒問暖還蠻不習慣的,也不能甚麼事都可以拜託父母幫忙而是要靠自己想辦法解決
但幸好室友人還不錯,常常一起揪出去吃飯或參加各類型講座,我也從他們身上學到如何獨立,讓我完全沒有感到孤單和焦慮,反而希望在中央大學裡可以認識更多像他們一樣的朋友!

**Comment by ds0w0** [comment]
我的室友人都很好，雖然一開始和我以往遇到的人差太多讓我覺得驚訝，但意外地相性很合，也聊了很多。btw宿舍沒有我想像中好睡，即便9A已經是很豪華的宿舍了，但我還是有時候會失眠，上大學是我人生中第一次外宿這麼久，需要時間適應也是難免。希望以後能夠越來越獨立自主，成為值得自豪的角色。

**Comment by chia-kai-chang** [comment]
1212121132131321321

**Comment by wcyeee** [comment]
我覺得上大學之後和高中最大的差別真的是有沒有住家裡。沒有住在家裡除了家事要自己處理之外，平常最常相處的家人也在18年後突然換成同齡人，住了18年的城市也可能換成另外一個沒去過幾次的城市。甚至上課模式也和之前高中國中國小完全不同，這個我覺得也是一個和之前很不一樣的點。雖然有許多不一樣，但我覺得只要好好照顧好自己，學會尊重別人，這些不習慣遲早都會變成我們的日常。

**Comment by Mminniehuang** [comment]
我自己覺得有三點上大學以後最不一樣的有三點，不過已經差不多都習慣了！
首先一樣是住宿，宿舍生活是我沒有過往經歷過的生活模式，真的需要很多的磨合和妥協，看著老舊斑駁的宿舍還會常常想家。
第二點是「去上課」。我覺得雖然每次要從宿舍到教室、或是從教室到另一間教室，其實路程並沒有想像中的遠，可是每當聽到50分催促腳步的鐘聲都會感到有些心累。而且因為總是匆匆來去，教室變成只是一個空間而已，沒有像國高中那樣小小的、溫馨的，更遑論歸屬感。
最後一點就是工讀。上大學想要變得更獨立一點，所以我找了工讀，用自己賺的微薄薪水當作生活費，不再跟爸媽拿。雖然生活變得有些緊湊，但我覺得很充實也很安心。
